### ISSUE TITLE
Missing properties from sampleConfig in Wiki 

### ISSUE THREAD

**Issue by Biboba** [Esri/arcgis-powershell-dsc#310]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Module Version

* 3.1.1

### Affected Resource(s)

* Wiki and samples config

### Expected Behavior

Samples configs should be using variables available in the Wiki.

### Actual Behavior

In [GISServer-GeneralPurpose-MultipleWA-MultiContext](https://github.com/Esri/arcgis-powershell-dsc/blob/master/SampleConfigs/v3/Gis%20Servers/GISServer-GeneralPurpose-MultipleWA-MultiContext.json) some properties used cannot be found in the [Wiki](https://github.com/Esri/arcgis-powershell-dsc/wiki/V3.-Variables-reference-page-for-JSON-configuration-files):

```
{
            "NodeName": "[Machine Name]",
            "Role": [
                "Server",
                "ServerWebAdaptor"
            ],
            "ServerContext":"[Node Server Context]",
            "AdminAccessEnabled": ["False","True"]
 }
```

I cannot find "ServerContext", "AdminAccessEnabled" properties in a node context. It can be found in the "configData" but not in the Node one. What are they used for ? How do they behave diffently ? I believe they should be explained (cf issue https://github.com/Esri/arcgis-powershell-dsc/issues/146) and at least listed !

### Steps to Reproduce

Check the [Wiki](https://github.com/Esri/arcgis-powershell-dsc/wiki/V3.-Variables-reference-page-for-JSON-configuration-files), in node section, properties "ServerContext" and "AdminAccessEnabled" are missing

Thanks !

**Comment by cameronkroeker** [comment]
@Biboba , thanks for the suggestion. We will work on getting these better defined within the Wiki, Thanks!

**AllNodes.ServerContext** and **AllNodes.AdminAccessEnabled** have the same meaning as **ConfigData.ServerContext** and **ConfigData.WebAdaptor.AdminAccessEnabled**:

**ServerContext** - The context of the Server's Web Adaptor.
**AdminAccessEnabled** - (Optional) This enables access to ArcGIS Server Manager and the ArcGIS Server Admin Directory. The default value is false.

The difference is **ConfigData.ServerContext** and **ConfigData.WebAdaptor.AdminAccessEnabled** are global parameters, meaning this will apply to all **AllNodes.Role.ServerWebAdaptor** that are defined in the json. Whereas,  **AllNodes.ServerContext** and **AllNodes.AdminAccessEnabled** are specific to each node. **AllNodes.ServerContext** and **AllNodes.AdminAccessEnabled** allow for having multiple Server WebAdaptors with different context names.

**Comment by cameronkroeker** [comment]
@Biboba,

These changes are now included in the v3.3.0 varaible reference wiki page:

https://github.com/Esri/arcgis-powershell-dsc/wiki/v3.3.0-Variables-reference-page-for-JSON-configuration-files

Thanks,
Cameron K.
